src/otx/core/data/dataset/anomaly.py

```diff
@@ -76,49 +69,35 @@ def __init__(
     def _get_item_impl(
         self,
         index: int,
-    ) -> AnomalyClassificationDataItem | AnomalySegmentationDataBatch | AnomalyDetectionDataBatch:
+    ) -> TorchDataItem:
         datumaro_item = self.dm_subset[index]
         img = datumaro_item.media_as(Image)
         # returns image in RGB format if self.image_color_channel is RGB
         img_data, img_shape, _ = self._get_img_data_and_shape(img)
+        img_data = to_dtype(to_image(img_data), dtype=torch.float32) / 255.0
 
         label = self._get_label(datumaro_item)
+        # TODO apply transforms
+        img_data = self.transforms(img_data)
+        # TODO test with masks and boxes
 
-        item: AnomalyClassificationDataItem | AnomalySegmentationDataItem | AnomalyDetectionDataItem
+        item: TorchDataItem
```

Comment:
thinking out loud for brainstorming here :)

`TorchItem`, `DataItem`, `Item`, `ImageItem`